lms/static/coffee/spec/instructor_dashboard/send_email_spec.coffee

```diff
@@ -0,0 +1,82 @@
+describe "Bulk Email Queueing", ->
```

Comment:
I'd like @scottrish to weigh in on that, as we've been trying to increase velocity.